Build failure on Windows

```
[INFO]   [msbuild]   D:\a\cyberduck\cyberduck\core\dll\Cyberduck.Core.ikvmproj : error MSB4236: The SDK 'IKVM.NET.Sdk' specified could not be found.
```

--- a/cli/osx/pom.xml
+++ b/cli/osx/pom.xml
@@ -228,17 +228,8 @@
         </dependency>
         <dependency>
             <groupId>org.fusesource.jansi</groupId>
-            <artifactId>jansi</artifactId>
+            <artifactId>libjansi</artifactId>
             <version>${jansi.version}</version>
-            <classifier>x86_64</classifier>
-            <type>jnilib</type>
-            <scope>runtime</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.fusesource.jansi</groupId>
-            <artifactId>jansi</artifactId>
-            <version>${jansi.version}</version>
-            <classifier>arm64</classifier>
             <type>jnilib</type>
             <scope>runtime</scope>
         </dependency>

